ports/liburing/portfile.cmake

```
@@ -0,0 +1,41 @@
+vcpkg_fail_port_install(MESSAGE "'${PORT}' is for Linux" ON_TARGET "windows" "uwp" "osx" "ios" "android")
```

```suggestion
vcpkg_fail_port_install(ON_TARGET "windows" "uwp" "osx" "ios" "android")
```
This message is not necessary.